README.md

```diff
@@ -1,5 +1,5 @@
 <a href="https://comit.network">
-<img src="logo.svg" height="120px">
+  <img src="logo.svg" height="120px" />
```

Comment:
Oh, my bad 😅

Reply:
added `alt` as well, will update the other READMEs accordingly, so we don't run into this problem at a different point again.